scripts/kubecf-apply.sh

```diff
@@ -3,6 +3,8 @@ source scripts/include/setup.sh
 
 require_tools helm kubectl
 
+: "${XIP_IO:=xip.io}"
```

Comment:
I believe the term use use [elsewhere](https://github.com/SUSE/cf-ci/blob/567eb68268846b22918eb13321b7bd8424d1fc58/qa-pipelines/tasks/clean_terraform_deploy.yml#L16) for this is `MAGIC_DNS_SERVICE`; would it be okay to reuse that for consistency?

Reply:
Absolutely; force-pushed an update.